### DISCUSSION TITLE
Ollama support windows now but the command does not work for windows

### DISCUSSION THREAD

**Discussion by EidosL** [logancyang/obsidian-copilot#312] [Q&A]
OLLAMA_ORIGINS=app://obsidian.md* ollama serve this command seems not working on windows.
Is anyone knows how to set it up on windows？

**Comment by logancyang** [comment]
I asked gpt-4 and it gave me the correct answer:

![image](https://github.com/logancyang/obsidian-copilot/assets/4860545/603f8934-389f-473c-8190-7130d5530b6c)

![image](https://github.com/logancyang/obsidian-copilot/assets/4860545/fa37c482-e836-45e7-be63-3cd1e162f761)

Just remember to quit the Ollama app first or you will see

![image](https://github.com/logancyang/obsidian-copilot/assets/4860545/b440f31c-4c39-45f4-b71a-de942aba5782)
